user_guide_src/source/extending/composer_packages.rst

```diff
@@ -0,0 +1,109 @@
+##########################
+Creating Composer Packages
+##########################
+
+You can make the :doc:`../general/modules` you create into Composer packages,
+or can create a Composer package for CodeIgniter 4.
+
+.. contents::
+    :local:
+    :depth: 2
+
+****************
+Folder Structure
+****************
+
+Here's a typical directory structure for a Composer package::
+
+    your-package-name/
+    ├── README.md
+    ├── composer.json
+    ├── src/
+    │   └── YourClass.php
+    └── tests/
+        └── YourClassTest.php
```

Comment:
I think `.gitignore` can be added so `vendor/` can be excluded so it "publish-ready" to packagist:

```
# .gitignore
/vendor
```

Reply:
Added. Thanks.